proc/types.go

```diff
@@ -24,13 +26,17 @@ func (dbp *Process) pointerTo(typ dwarf.Type) dwarf.Type {
 
 func (dbp *Process) findTypeExpr(expr ast.Expr) (dwarf.Type, error) {
 	dbp.loadPackageMap()
+	if lit, islit := expr.(*ast.BasicLit); islit && lit.Kind == token.STRING {
```

Comment:
Would be helpful to have some documentation around why we expect a string literal here.